Content.Shared/Construction/MaterialConstructionGraphStep.cs

```diff
@@ -12,30 +17,34 @@ public class MaterialConstructionGraphStep : EntityInsertConstructionGraphStep
     {
         // TODO: Make this use the material system.
         // TODO TODO: Make the material system not shit.
-        public StackType Material { get; private set; }
+        private string MaterialPrototypeId { get; [UsedImplicitly] set; } = default!;
+
+        public StackPrototype MaterialPrototype =>
+            IoCManager.Resolve<IPrototypeManager>().Index<StackPrototype>(MaterialPrototypeId);
+
         public int Amount { get; private set; }
 
         public override void ExposeData(ObjectSerializer serializer)
         {
             base.ExposeData(serializer);
 
-            serializer.DataField(this, x => x.Material, "material", StackType.Metal);
+            serializer.DataField(this, x => x.MaterialPrototypeId, "material", "Steel");
             serializer.DataField(this, x => x.Amount, "amount", 1);
         }
 
         public override void DoExamine(FormattedMessage message, bool inDetailsRange)
         {
-            message.AddMarkup(Loc.GetString("Next, add [color=yellow]{0}x[/color] [color=cyan]{1}[/color].", Amount, Material));
+            message.AddMarkup(Loc.GetString("Next, add [color=yellow]{0}x[/color] [color=cyan]{1}[/color].", Amount, MaterialPrototypeId));
```

Comment:
Same here, should use a human-readable name

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3387/commits/e8560e28a76adc8b9f6b32f0e28c8d58c580c00f